spartan/aztec-chaos-scenarios/values.yaml

```diff
@@ -1,3 +1,6 @@
+nameOverride: null
+fullnameOverride: null
+
 global:
   # When deploying, override the namespace to where spartan will deploy to, this will apply all chaos experiments to all pods within that namespace
   # run deployment with --values global.namespace=your-namespace
```

Comment:
Nice, thanks for including this note. It wasn't clear to me at first why the values files used different namespace refs.